pusher-api/api-reference.md

```diff
@@ -0,0 +1,392 @@
+</details>
+
+<details>
+<summary>Click to see response attributes.</summary><br/>
+<p>The service returns a <code>200 OK</code> response without any content.</p>
+</details>
+
+
+## Delete a webhook subscription
+
+Deletes an existing webhook subscription.
+
+```
+DELETE /organizations/{organizationUuid}/subscriptions/{subscriptionUuid}
+```
+
+See [Delete a subscription example](#delete-a-subscription).
+
+### Parameters
+
+<details>
+<summary>Click to see all request parameters for deleting a subscription.</summary>
+
+|Name |Type |In |Required/Optional |Description
+|---- |---- |---- |---- |----
+|organizationUuid |string |path |required |Unique identifier for your organization. You can use following options to fill in this value: <br/><ul><li> Get it by using the https://oauth.izettle.com/users/me endpoint of OAuth2 API. See [OAuth2 API](https://github.com/iZettle/api-documentation/blob/master/authorization.adoc) for more information.</li><li> Use `self` as the value. This will retrieve your organizationUuid through the authentication information in the request.</li></ul>
+|subscriptionUuid |string |path |required |Unique identifier for an existing subscription as UUID version 1.
+</details>
+
+
+### Responses
+<details>
+<summary name="deleteHttpStatusCode">Click to see HTTP status codes.</summary>
+
+|Status code |Description
+|---- |----
+|204 No Content| Returned when the service deletes the subscription successfully.
+|401 Unauthorized |Returned when one of the following occurs: <br/><ul><li> The authentication information is missing in the request.</li><li>The authentication token has expired.</li><li>The authentication token is invalid.</li></ul>
+|404 Not Found|Returned when one of the following occurs: <br/><ul><li>The ```subscriptionUuid``` is missing in the request.</li><li>A subscription was not found for the ```subscriptionUuid``` passed in the request</li></ul> 
+|500 Internal Server Error| Returned when the service encounters an internal server error. In case this error persists, contact [support](mailto:api@zettle.com).
+</details>
+
+<details>
+<summary>Click to see response attributes.</summary><br/>
+<p>The service returns a <code>204 No Content</code> response without any content.</p>
+</details>
+
+## Supported events
+
+<details>
+<summary>Click to see a list all the events supported by the service and their corresponding authorization scopes.</summary>
+
+|Event name |Required scope | Description
+|---- |---- |----
+|PurchaseCreated|READ:PURCHASE|Triggered when a new purchase is created.
+|InvoiceCreated|READ:FINANCE|Triggered when a new invoice is created.
+|OrganizationUpdated|READ:USERINFO|Triggered when information of an organization is updated.
+|OrganizationFeatureUpdated|READ:USERINFO|Triggered when an organization feature set has been updated.
+|PaymentInitiated|READ:FINANCE|Triggered when a new payment has been initiated.
```

Comment:
Don't think this is descriptive enough. Reading it i don't understand the actual trigger.